### PR TITLE
Automate the epsilon picking based on percentile of model values for …

### PR DESCRIPTION
**Pull request by fourndo**
Automate the epsilon picking based on percentile of model values for DEFAULT mode. Fix example.

Fix bug with Maps using array of values.

### CODE REVIEW

File: SimPEG/Directives.py [hunk 1]
```diff
@@ -343,14 +356,14 @@ def endIter(self):
...
 
-            # Cool the threshold parameter if required
-            if getattr(self, 'factor', None) is not None:
-                eps = self.reg.eps / self.factor
-
-                if getattr(self, 'eps_min', None) is not None:
-                    self.reg.eps = np.max([self.eps_min,eps])
-                else:
-                    self.reg.eps = eps
+#            # Cool the threshold parameter if required
```
Comment (lheagy) [inline SimPEG/Directives.py]: do you want to remove the commented-out code? or is this in progress?

File: SimPEG/Maps.py [hunk 2]
```diff
@@ -502,7 +502,9 @@ def __init__(self, mesh, indActive, valInactive, nC=None):
         if Utils.isScalar(valInactive):
             self.valInactive = np.ones(self.nC)*float(valInactive)
         else:
-            self.valInactive = valInactive.copy()
+            self.valInactive = np.ones(self.nC)
```
Comment (lheagy) [inline SimPEG/Maps.py]: what was the bug here?

Comment (fourndo) [inline SimPEG/Maps.py]: if the number of valInactive is different than nC, we were getting an error.

File: SimPEG/Directives.py (hunk 1 above)
Comment (fourndo) [inline SimPEG/Directives.py]: We might want to explore the cooling schedule later when doing the non-linear stuff. So lets keep it for now.

Comment (lheagy) [inline SimPEG/Directives.py]: sounds good!

File: SimPEG/Maps.py (hunk 2 above)
Comment (lheagy) [inline SimPEG/Maps.py]: ah, ok.

**Comment by lheagy** [comment]
looks good to me!

**Comment by lheagy** [comment]
@fourndo : feel free to merge if you are happy with this

**Pull request merged**